packages/edit-site/src/components/site-hub/style.scss

```
@@ -23,18 +23,55 @@
 	}
 }
 
-.edit-site-site-hub__title {
+.edit-site-site-hub__title .components-button {
+	color: $gray-200;
+	display: block;
+	flex-grow: 1;
+	font-size: 15px;
+	font-weight: 500;
+	overflow: hidden;
+	// Add space for the ↗ to render.
+	padding-right: $grid-unit-20;
+	position: relative;
+	text-decoration: none;
 	text-overflow: ellipsis;
 	white-space: nowrap;
-	overflow: hidden;
-	flex-grow: 1;
-	color: $gray-200;
-}
 
-.edit-site-site-hub__site-view-link {
-	margin-right: var(--wp-admin-border-width-focus);
-	svg {
-		fill: $gray-200;
+	&:hover,
+	&:focus,
+	&:active {
+		color: $gray-200;
+	}
+
+	&:focus {
+		// Defer to :focus-visible below.
+		box-shadow: none;
+		outline: none;
+	}
+
+	&:focus-visible {
+		// Push the shadow away from the title.
+		box-shadow: 0 0 0 var(--wp-admin-border-width-focus) $gray-900, 0 0 0 calc(2 * var(--wp-admin-border-width-focus)) var(--wp-admin-theme-color);
+
+		// Windows High Contrast mode will show this outline, but not the box-shadow.
+		outline: 2px solid transparent;
+		outline-offset: 2px;
+	}
+
+	&::after {
+		content: "↗";
```

```suggestion
		content: "\2197";
```

Just to be safe, I think Unicode is better.

---

> Is there a reason to use focus-visible?

I would think it's unexpected to have the focus visible if it's not necessary, since it has already been interacted with. This aligns with UX principles by ensuring that visual cues like focus outlines are only used when they add value to the interaction.

---

The reason I suggested using the `:focus` pseudo-element is to match the mouse click UX in the header.

The site icon and the command palette button get outlined while you click them with the mouse, but the title link doesn't. What do you think about this?


https://github.com/WordPress/gutenberg/assets/54422211/d1c47b85-67fd-49cd-8f48-297d824c1275